### ISSUE TITLE
`opts` seemingly ignored when `exts` is specified

### ISSUE THREAD

**Issue by subsetpark** [pyrmont/markable#4]
Here's an example failing test:

```
(deftest render-footnote
  (let [footnote ```
                 foo[^bar]

                 [^bar]: baz
                 ```
        expected "<p>foo<sup class=\"footnote-ref\"><a href=\"#fn1\" id=\"fnref1\">1</a></sup></p>\n<section class=\"footnotes\">\n<ol>\n<li id=\"fn1\">\n<p>baz <a href=\"#fnref1\" class=\"footnote-backref\">\xE2\x86\xA9</a></p>\n</li>\n</ol>\n</section>\n"
        rendered (bagatto/markdown->html footnote)]
    (is (= expected rendered))))
```

When the invocation is `(markable/markdown->html (string md) [:footnotes])`, the test passes.

When the invocation is `(markable/markdown->html (string md) [:footnotes] [])`, the test fails:

```
> Failed: render-footnote
Assertion: (= expected rendered)
Expect: "<p>foo<sup class=\"footnote-ref\"><a href=\"#fn1\" id=\"fnref1\">1</a></sup></p>\n<section class=\"footnotes\">\n<ol>\n<li id=\"fn1\">\n<p>baz <a href=\"#fnref1\" class=\"footnote-backref\">\xE2\x86\xA9</a></p>\n</li>\n</ol>\n</section>\n"
Actual: "<p>foo<a href=\"baz\">^bar</a></p>\n"
```

**Comment by pyrmont** [comment]
This is a bit embarrassing but when I added support for an extensions tuple as the third argument, I forgot to update the `argc` check to be `>=` rather than just `==`. It should be fixed as of b7880c98a30e1c0b76bfc2d675b7aabe6f82d231.

**Comment by subsetpark** [comment]
:+1: !
